pyproject.toml

```diff
@@ -155,6 +155,7 @@ doc8 = { version = "*", optional = true }            # groups = ["dev", "all"]
 readme-renderer = { version = "*", optional = true } # groups = ["dev", "all"]
 tabulate = { version = "*", optional = true }        # groups = ["dev", "all", "doctests"]
 pyoxidizer = { version = "*", optional = true }      # groups = ["dev", "all", "build"]
+interpolation = "^2.2.4"
```

Comment:
Make sure it's included as an optional dependency :)

Reply:
sorry, why optional? I expect that, assuming this offers sufficient improvement, there'll be no need to maintain the scipy interpolator version.

Is it because the dependency is specific to gamma?

Reply:
Every dependency in PyMedPhys is set to optional. That way, if someone just wants to set up a minimal environment, say an icom listener on a raspberry pi with absolute minimal dependencies, then they can just install the absolute bare minimum. Quite important when something is being installed in a sensitive location. Potentially a vendor might want to just utilise a small component of pymedphys, and they just want to get that component only, and only that small subset of dependencies only.

A massively minimal install is something that is explicitly supported by pymedphys.

Reply:
> A **massively minimal** install

Entertaining juxtaposition there 😆 

I understood the minimal install point, I just wasn't sure/never checked how aggressively we were supporting that (e.g., I wasn't aware until looking now that *every* dependency is optional).